lms/static/styles/components/_Dialog.scss

```diff
@@ -66,8 +66,6 @@
 
   .Button {
     @include focus.outline-on-keyboard-focus;
-    &:not(:first-child) {
-      margin-left: 5px;
-    }
+    margin-left: 5px;
```

Comment:
It's not perfect, but it also It does not hurt to render the margin on the first button. This rule was causing a problem with the submit button since it was nested in a new parent wrapper. The 5px margin was only being applied after the validation message opened. The super simple fix is to just apply the rule on all action buttons in the dialog. I think the most we ever see is 3 buttons IIRC.